Makefile

```diff
@@ -64,6 +64,10 @@ $(OFILES) $(MOFILE): $(HFILES)
 clean:
 	rm -f *.o $(CLEANFILES)
 
+.PHONY: fclean
+fclean: clean
```

Comment:
Let's undo the changes to this file. If you want to tweak the build rules, it can happen in a separate pull request.